paddle/operators/beam_search_op.cc

```diff
@@ -48,12 +48,20 @@ void BeamSearch::operator()(const framework::LoDTensor &pre_ids,
   size_t low_offset = 0;
   for (auto &items : selected_items) {
     low_level.push_back(low_offset);
+    sort(items.begin(), items.end(), [](Item &a, Item &b) {
```

Comment:
don't we need to sort by score?

Reply:
no, the scores just need to filter and get the top N.

sort or not sort score will not change the result candidates.